glean-core/src/metrics/uuid.rs

```diff
@@ -59,10 +59,9 @@ impl UuidMetric {
     /// # Arguments
     ///
     /// * `glean` - The Glean instance this metric belongs to.
-    pub fn generate_and_set(&self, storage: &Glean) -> Uuid {
+    pub fn generate_and_set(&self, storage: &Glean) {
```

Comment:
If we do this then this API suddenly becomes different with respect to the other bindings: we should either restore it here or change the other bindings, too (filing a bug should be ok if we don't have one already)

cc @badboy

Reply:
Oops, right, I'd forgotten I'd only filed https://bugzilla.mozilla.org/show_bug.cgi?id=1673017 and it hadn't been processed/adopted. I definitely need to put this back for now.

I guess my Opinions were too strong when I wrote it. : )